### PR TITLE
Add draft tests for human ship placement

### PR DESCRIPTION
**Pull request by kbergstrom78**
Add tests for:

- Human can place ships in valid locations
- User can enter valid sequences to place both ships
- Entering invalid ship placements prompts user to enter valid placements

### CODE REVIEW

File: spec/game_spec.rb [hunk 1]
```diff
@@ -37,16 +37,22 @@
...
+      @game.human_place_ship(@game.human_cruiser, ["A1", "A2", "A3"])
+      expect(@game.human_board.cells["A1"].ship).to be_a(Ship)
     end
 
     it 'takes coordinates from user to place ships' do
-
+      expect(@game.human_board.ships).to eq([])
+      @game.human_place_ship(@game.human_cruiser)
+      @game.human_place_ship(@game.human_submarine)
+      expect(@game.human.ships).to eq([@game.human_cruiser, @game.human_submarine])
```
Comment (kaileykaes) [inline spec/game_spec.rb]: What is the game.human method doing?

Comment (kaileykaes) [reply spec/game_spec.rb]: Oh, is this meant to be @game.human_board.ships?

File: spec/game_spec.rb [hunk 2]
```diff
@@ -37,16 +37,22 @@
       expect(@game.robo_board.ships).to eq([@game.robo_cruiser, @game.robo_submarine])
     end
 
-    it 'human places ships randomly in valid locations' do
-    
+    it 'human places ships in valid locations' do
+      @game.human_place_ship(@game.human_cruiser)
+      @game.human_place_ship(@game.human_submarine)
+      @game.human_place_ship(@game.human_cruiser, ["A1", "A2", "A3"])
+      expect(@game.human_board.cells["A1"].ship).to be_a(Ship)
```
Comment (kaileykaes) [inline spec/game_spec.rb]: I think I'd suggest testing every cell a ship is meant to be in and also testing that the same ship is in every cell here.

File: spec/game_spec.rb (hunk 1 above)
Comment (kbergstrom78) [reply spec/game_spec.rb]: Ugh - yes, game.human_board.ships.  Good catch.  I'll update

File: spec/game_spec.rb (hunk 2 above)
Comment (kbergstrom78) [reply spec/game_spec.rb]: Got it - I'll work on it.  Thanks!

**Review by kaileykaes — APPROVED** [review]
Changes look good.

**Pull request merged**